examples/llm/src/models/mosaic_gpt/configuration_mosaic_gpt.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         fan_mode: str = 'fan_in',
         init_nonlinearity: str = 'relu',
         embedding_fraction: float = 1.0,
-        low_precision_layernorm: bool = True,
+        norm_type: str = 'low_precision_layernorm',
```

Comment:
should we accept the old arg and just map it to the correct norm type for backwards compat reasons?

Reply:
what you're describing is just a config updt; I wouldn't worry about it.


Changing this
```
        self.ln_1 = layernorm_class(d_model, device=device)
```
to this
```
        self.norm_1 = norm_class(d_model, device=device)
```
is the bigger bwd compat issue.
(param names from old models will be diff; cant just load them)

Reply:
that is fair